controllers/admin/AdminImportController.php

```diff
@@ -3643,6 +3646,10 @@ public function attributeImport($offset = false, $limit = false, &$crossStepsVar
             $info = array_map('trim', $info);
 
             try {
+                if ($truncateOnlyProductCombinations) {
+                    $this->deleteCombinationsByProductId($info['id_product'], $processedProducts);
+                }
```

Comment:
Unfortunately this won't work. The process of big files is split into multiple separate request, each of them processing 5 rows at a time. Your `$processedProducts` array will be empty at the start of each request.

You need to, somehow, make this $processedProducts persistent. Maybe use `$crossStepsVariables`